src/userApp/emailValidation.js

```diff
@@ -0,0 +1,13 @@
+import { body } from 'express-validator';
+
+const validateEmail = () => [
+	body('email')
+		.isEmail()
+		.withMessage('email_invalid')
+		.notEmpty()
+		.withMessage('email_required'),
+
+	body('email', 'email_required').notEmpty(),
```

Comment:
as Fabrice mentioned you should remove the second validation check is you already checked that it is not empty